src/content/reference/react-dom/hooks/useFormState.md

```diff
@@ -0,0 +1,219 @@
+---
+title: useFormState
+canary: true
+---
+
+<Canary>
+
+The `useFormState` Hook is currently only available in React's canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+<Intro>
+
+`useFormState` is a Hook that allows you to show error messages or other responses after submitting a form.
+
+```js
+const [state, formAction] = useFormState(action, initalState);
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `useFormState()` {/*useformstate*/}
+
+Call `useFormState` at the top level of your component to see the return value of an action after submitting a form. You pass `useFormState` an existing action as well as an initial state, and it returns a new action that you use when submitting your form, along with the latest form state.
```

Comment:
We haven't really established what a `action` is at this point. Could you rewrite this to avoid the term action or add a comment that we need to link to the actions documentation once we have it so we remember to come back and add the link?

Reply:
In this case, isn't `action` referring to the <form action={action}>?

Reply:
I added a sentence about it and linked to the <form> docs for more context, let me know what you think.